pkg/interface/src/views/components/GroupSearch.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useMemo, useCallback } from "react";
```

Comment:
Why have you duplicated `InviteSearch` with restricted features? You can enable or disable both groups and members from `InviteSearch` with booleans. I believe the previous "new page" did so, too.

Reply:
Presumably because of the way it looks; but you could just update invitesearch to use indigo ... which, I suppose, would create inconsistent UI in the other apps. Hm.

Reply:
largely to get an indigo-based dropdown component, yeah. I also question the UX of jamming group associations and ship invites into the one field tbh.

Reply:
I believe holdover from OS1 development where our apps were loose as to whether you were inviting a group or a ship; I can't think of any current cases that let you search both at the same time atm. So maybe we could eliminate InviteSearch if we had a GroupSearch alongside it. @urcades? 

Just something to think about I guess.

Reply:
Yeah, I'm with Liam in desiring a more focused state in this interface in general. With the emphasis we place on group-oriented module usage, it'd make sense to simplify in this case.

Reply:
> I also question the UX of jamming group associations and ship invites into the one field tbh.

He's got a point here too, this original UX was really only done " " out of necessity " " to afford for the nuance we allowed at the time.